modules/api-import-export/src/main/java/org/wso2/carbon/apimgt/importexport/utils/APIImportUtil.java

```diff
@@ -235,8 +235,9 @@ public static void importAPI(String pathToArchive, String currentUser, boolean i
             allMatchedApis = provider.searchAPIs(importedApi.getId().getApiName(), "Name", null);
             //if an API exist with the same name
             if (!allMatchedApis.isEmpty()) {
-                for (API matchAPI : allMatchedApis) {
-                    if (matchAPI.getId().getVersion().equalsIgnoreCase(importedApi.getId().getVersion())) {
+                for(API matchAPI:allMatchedApis) {
```

Comment:
Shall we fix the formatting here?

Reply:
@tharikaGitHub: Thanks for the comment. Fixed the formatting issue